src/shmem_env_defs.h

```diff
@@ -105,7 +105,5 @@ SHMEM_INTERNAL_ENV_DEF(MPI_THREAD_LEVEL, string, "MPI_THREAD_SINGLE", SHMEM_INTE
                        "Specify the MPI threading level when MPI is used as the process manager")
 #endif
 
-#ifdef USE_BT
-SHMEM_INTERNAL_ENV_DEF(BACKTRACE_METHOD, string, "execinfo", SHMEM_INTERNAL_ENV_CAT_OTHER, 
+SHMEM_INTERNAL_ENV_DEF(BACKTRACE_METHOD, string, "auto", SHMEM_INTERNAL_ENV_CAT_OTHER, 
```

Comment:
I think shortening this to `SHMEM_BACKTRACE` sounds like an env variable used to enable/disable backtrace. However, we are trying to use this variable for choosing the method user wants, right? 

If we choose "disabled" the default, then user will see no backtraces unless they set this parameter. I can see then why you would like `SHMEM_BACKTRACE` as opposed to `SHMEM_BACKTRACE_METHOD`. We can go with this then if you guys think this would be ideal.